test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests.cs

```diff
@@ -236,7 +214,7 @@ public SamplesEndToEndTests(TestFixture fixture)
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
-        [Fact(Skip = "Needs investigation")]
+        [Fact(Skip = "Node language worker isn't handling duplicate query params properly")]
```

Comment:
I debugged into this, and even with my fixes for properly deduping query params, node is not doing so. I verified I'm passing the correct query param binding data to the language worker.

I think the language worker is handling the request params itself? @mhoeger ?

Reply:
Looking into this and will open a node worker bug if appropriate. Thanks!